network/Cargo.toml

```diff
@@ -15,13 +15,10 @@ serde = "1.0"
 serde_derive = "1.0"
 ckb-util = { path = "../util" }
 stop-handler = { path = "../util/stop-handler" }
-unsigned-varint = {git = "https://github.com/paritytech/unsigned-varint", features = ["codec"]}
-lru-cache = { git = "https://github.com/nervosnetwork/lru-cache", rev = "b36a4d1" }
 log = "0.4.5"
 bytes = "0.4.12"
 tokio = "0.1.18"
 futures = "0.1"
-snap = "0.2"
```

Comment:
used on #859

Reply:
> used on #859

You can add it back when your PR merged.

This PR can merge if it passed the CI.